numpyro/hmc_util.py

```diff
@@ -726,7 +726,8 @@ def initialize_model(rng, model, *model_args, init_strategy='uniform', **model_k
     within their respective support.
 
     :param jax.random.PRNGKey rng: random number generator seed to
-        sample from the prior.
+        sample from the prior. The returned `init_params` will have the
```

Comment:
Would suggest adding a test for this parametrized on `init_strategy` to make sure that we end up returning distinct initial points for either strategy.

Reply:
Nice suggestion! We want to make sure that we get a fixed init_params for each random seed.